File: python/nivel 5 simulacion/26.py
"""
Ejercicio 26
Simular un login (usuario y contraseña correctos)
"""
# Credenciales
usuario_correcto = "admin"
password_correcta ="1234"

# Datos ingresados
usuario = input("Ingresa el Usuario: ")
password = input("Ingresar la contraseña: ")

# Validación
if usuario == usuario_correcto and password == password_correcta:
  print("Acceso Consedido")
else:
  print("Acceso Denegado")

File: javascript/nivel 5 simulacion/26.js
/*
Ejercicio 26
Simular un login (usuario y contraseña correctos)
*/

//credenciales
let usuarioCorrecto = "admin";
let passwordCorrecto ="1234";

//datos ingresados
let usuario = prompt("Ingresa el usuario: ");
let password = prompt("ingresa la contraseña: ");

//validacion
if(usuario == usuarioCorrecto && password == passwordCorrecto){
  console.log("Aceeso Permitido");
}else{
  console.log("Aceeso Denegado");
}